modules/dmrpp_module/DmrppArray.cc

```diff
@@ -3088,7 +3088,7 @@ unsigned int DmrppArray::buf2val(void **val){
         memcpy(*val, str_buf.data(), buf_size);
         return buf_size;
     }
-    return Vector::buf2val(val);
+    return Vector::buf2val_ll(val);
```

Comment:
Let's change this just a bit (the fix is correct, but to make it easier to read, put the `return Vector::buf2val_ll(val);` in an else block and make the case to unsigned int explicit:

else {
    return (unsigned int)Vector::buf2val_ll(val);
}

Make sure I'm not off my rocker with that and merge if it passes tests on your machine.

Thanks!

Reply:
Second  James' suggestion.